README.md

```diff
@@ -1,8 +1,14 @@
 # OpenDP
 [![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![Python](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue)](https://www.python.org/)
-[![ci tests](https://github.com/opendp/opendp/actions/workflows/smoke-test.yml/badge.svg)](https://github.com/opendp/opendp/actions/workflows/smoke-test.yml?query=branch%3Amain)
+
+[![Python](https://img.shields.io/badge/Python-3.8%20%E2%80%93%203.12-blue)](https://www.python.org/)
+[![R](https://img.shields.io/badge/R-grey)](https://www.r-project.org/)
+[![Rust](https://img.shields.io/badge/Rust-grey)](https://www.rust-lang.org/)
+
+[![main CI](https://github.com/opendp/opendp/actions/workflows/smoke-test.yml/badge.svg)](https://github.com/opendp/opendp/actions/workflows/smoke-test.yml?query=branch%3Amain)
+[![nightly CI](https://github.com/opendp/opendp/actions/workflows/nightly.yml/badge.svg)](https://github.com/opendp/opendp/actions/workflows/nightly.yml?query=branch%3Amain)
+[![links CI](https://github.com/opendp/opendp/actions/workflows/weekly-doc-check.yml/badge.svg)](https://github.com/opendp/opendp/actions/workflows/weekly-doc-check.yml?query=branch%3Amain)
```

Comment:
```suggestion
```

Reply:
Are you hesitant because it's just clutter? Or because failures will reflect poorly on the project, and it can be flakey?

I think it's useful to have all our checks in one place, the visibility to non-devs is actually a good thing, and at least one dev (me) hadn't thought to subscribe to notifications.

Maybe talk through this at next stand up?

Reply:
Ah: Let me move it to the workflows README: It's off the front page, so less of a distraction, but it's also more visible than it is now. Will make the change.